pallets/rc-migrator/src/lib.rs

```diff
@@ -136,6 +136,9 @@ pub const MAX_XCM_SIZE: u32 = 50_000;
 /// and Asset Hub.
 pub const MAX_ITEMS_PER_BLOCK: u32 = 1600;
 
+/// The maximum number of XCM messages that can be sent in a single block.
+pub const MAX_XCM_MSG_PER_BLOCK: u32 = 20;
```

Comment:
This is the resolution that we check with? So if our limit is 50 and this is set to 20 then we can at most overshoo by 20 and have 70 messages in the queue?